src/app/views/app-sections/StatusMessages.tsx

```diff
@@ -51,16 +52,31 @@ export function statusMessages(queryState: any, sampleQuery: IQuery, actions: an
       message = replaceLinks(status);
       urls = convertArrayToObject(extractedUrls);
     }
+    const isRSC = (permission: IPermission) => {
+      return RSC_PERMISSIONS_ENDINGS.some((ending) => permission.value.indexOf(ending) !== -1)
+    }
+
+    const filterPermissionsForRSC = () => {
+      return permissionModeType === DISPLAY_APPLICATION_PERMISSIONS ? permissions.filter(isRSC) : permissions;
+    }
+
+    const permissions: IPermission[] = scopes.hasUrl ? scopes.data : [];
+    const filteredScopes = filterPermissionsForRSC();
+    const noPerms = filteredScopes.length === 0;
 
     return (
-      <MessageBar messageBarType={messageType}
+      <MessageBar messageBarType={noPerms ? 5 : messageType}
```

Comment:
Is there a particular reason it's hard coded with a 5?

Reply:
for `messageBarType`, 5 shows the yellow card: 
![image](https://user-images.githubusercontent.com/55033656/124488570-dd0cbe00-dd64-11eb-94f3-154a7c55b273.png)

but i should of made it a constant